k8s.gcr.io/lib.sh

```diff
@@ -195,7 +200,33 @@ function empower_gcr_admins() {
         "${bucket}"
 }
 
-# Grant write privileges to a group
+# Grant full privileges to GCS admins
+# $1: The GCP project
+# $2: The bucket
+function empower_gcs_admins() {
+    if [ $# -ne 2 -o -z "$1" -o -z "$2" ]; then
+        echo "empower_gcs_admins(project, bucket) requires 2 arguments" >&2
+        return 1
+    fi
+    project="$1"
```

Comment:
Technically we don't need the project, but it feels inconsistent not to require it (IMO)